general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -236,15 +239,51 @@ def _accept_terms_of_use(self, user_input: str) -> str:
         """
         return self.post_request("/accept_terms_of_use", {"user_input": user_input})
 
-    def get_targets(self) -> Dict[str, Dict[str, List[str]]]:
+    def get_targets(self, simulator: Optional[bool] = None, **kwargs: Any) -> List[TargetInfo]:
```

Comment:
Sure, can do though wouldn't list the supported `kwargs` in the docstring count as documentation for the user? It can be easier to have dedicated flags but I guess my initial motivation was that if more flags were added on the server side, we wouldn't need to worry as much client-side (aside from docstring documentation)